2016/proposals/filip-ter.md

```diff
@@ -0,0 +1,197 @@
+##Providing sparse data structures as an implementation detail of the core and enhancing their functionality
+
+---
+
+###Table of Contents
+
+
+ 1. Abstract
+ 2. Project Goals
+ 3. Technical Details
+ 4. Milestones / Schedule of Deliverables
+ 5. Future Works
+ 6. Open Source Development Experience
+ 7. Academic Experience
+ 8. Why this project
+ 9. About Me
+ 10. Appendix
+ 
+---
+
+###Abstract
+
+Currently sparse data structures provided by pandas exist in the separate `sparse` module. This package has few users and is easy to overlook when developing new features due to its separation from the `core` module. I propose to change sparse into an implementation detail of the core data structures so that a user does not need to import different packages or classes if they want to use sparse functionality. 
```

Comment:
I don't think this is easy/possible. The ``sparse`` provides an underlying impl detail. I don't see an easy way to eliminate this. I suppose its possible, but this would be a very large change I think.